R/osmactive.R

```diff
@@ -478,6 +478,13 @@ classify_cycle_infrastructure_scotland = function(
         )
       )
   }
+  osm_classified = osm_classified |>
+    dplyr::mutate(
+      cycle_segregation = dplyr::case_when(
+        cycle_segregation == "Shared Footway" & distance_to_road > min_distance ~ "Off Road Path",
```

Comment:
An alternative would be to make *all* segments 'Off road path' if they are far from roads but I don't think that is needed.